js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,158 @@ mkdir my-new-bucket</pre>
       the Cloud Tools for PowerShell module via <code>Import-Module GoogleCloud</code>.
     </p>
   </div>
+  
+    <!-- Google Cloud BigQuery -->
+  <div ng-switch-when="google-cloud-bigquery">
+    <p>
+      Google BigQuery is a versatile tool that solves the problem of storing and querying massive 
+      datasets without having to worry about data formats, underlying resources, and other things 
+      that distract you from your analysis.
+    </p>
+    
+    <h2>Dataset</h2>
+    <p>
+      To use BigQuery in a Cloud project, first create a <em>Dataset</em> using the 
+      <cmdlet-ref name="New-BqDataset"></cmdlet-ref> cmdlet. This will take in basic information and 
+      create the resource serverside. Locally, a reference Dataset object is returned. To get a reference
+      object for an existing dataset, use <cmdlet-ref name="Get-BqDataset"></cmdlet-ref>.
+    </p>
+    <pre>
+# Makes a new dataset with DatasetId "my_dataset".
+$dataset = New-BqDataset "my_dataset" -Name "Test Dataset" -Description "Some test data for BigQuery"
+
+# The following two lines do the same thing.
+$dataset = Get-BqDataset "my_dataset"
+$dataset = $dataset | Get-BqDataset
+    </pre>
+    <p>
+      This object <em>$dataset</em> can be modified and passed into further cmdlets such as 
+      <cmdlet-ref name="Set-BqDataset"></cmdlet-ref> to manipulate cloud resources. This cmdlet also
+      handles adding and removing labels with <code>-SetLabel</code> and <code>-ClearLabel</code>.
+    </p>
+    <pre>
+# Updates the Name field of $dataset.
+$dataset.Name = "A Different Name"
+$dataset = Set-BqDataset "my_dataset"
```

Comment:
I feel like you can just run `Set-BqDataset "my_dataset"` without assigning it to a variable since you are not really using it anywhere in this example?